sunpy/instr/goes.py

```diff
@@ -52,8 +61,8 @@ def get_goes_event_list(timerange, goes_class_filter=None):
     # use HEK module to search for GOES events
     client = hek.HEKClient()
     event_type = 'FL'
-    tstart = timerange.start
-    tend = timerange.end
+    tstart = timerange.start()
+    tend = timerange.end()
```

Comment:
This is a regression? Recent (ish) updates to `TimeRange` made `start` and `end` properties?